presidio-anonymizer/presidio_anonymizer/operators/custom.py

```diff
@@ -0,0 +1,40 @@
+"""Replaces the PII text with function result."""
+from typing import Dict
+
+from presidio_anonymizer.operators import Operator, OperatorType
+from presidio_anonymizer.services.validators import validate_type
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class Custom(Operator):
+    """
+    Replace PII text entity with the results of a function executed on the PII text.
+
+    The function retrun type must be a string    
+    """
+
+    NEW_VALUE = "new_value"

```

Comment:
"new_val" perhaps LAMBDA = "lambda" ?

Reply:
Also vote for a name change, either lambda, function, callback...

Reply:
changed parameter name to lambda